R/AbstractGraphReporter.R

```diff
@@ -222,7 +222,10 @@ AbstractGraphReporter <- R6::R6Class(
               } else if (is.numeric(colorFieldValues)) {
                 # If numeric field, assume continuous
                 newPallete <- grDevices::colorRamp(colors = colorFieldPallete)
-                plotDTnodes[, color := newPallete(get(colorFieldName))]
+                plotDTnodes[!is.na(get(colorFieldName)), scaledColorValues := get(colorFieldName) / max(get(colorFieldName))]
```

Comment:
can you please add comments on each line explaining what's happening here? It's obvious that the ordering of these three lines matters, but I had to stare at it for a few mins to figure out what was happening